### PR TITLE
Feedback screen

### PR DESCRIPTION
**Pull request by gustafguner**
This PR adds the feedback screen. When reporting a bug via email, a text file `device-info.txt` will be attached automatically.

`device-info.txt` example:
```
Device: iPhone X
System: iOS 12.0
```

### CODE REVIEW

File: Src/Screens/ProfileScreen/Screens/FeedbackScreen/Feedback.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//  Copyright © 2019 Hedvig AB. All rights reserved.
+//
+
+import Flow
+import Form
+import Presentation
+import UIKit
+
+struct Feedback {
+    let presentingViewController: UIViewController
```
Comment (Iamsamwhoami) [inline Src/Screens/ProfileScreen/Screens/FeedbackScreen/Feedback.swift]: As this is a presentable, presentingViewController will be itself, so remove this and on line 39 just use viewController.

File: Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/FeedbackLabel.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+struct FeedbackLabel {}
+
+extension FeedbackLabel: Viewable {
+    func materialize(events: ViewableEvents) -> (UIView, Disposable) {
+        let bag = DisposeBag()
+        let containerView = UIView()
+        
+        let backgroundWithLabel = BackgroundWithLabel(
+            labelText: DynamicString("Hjälp oss bli\nbättre!"),
```
Comment (Iamsamwhoami) [inline Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/FeedbackLabel.swift]: Add this to GraphCMS

File: Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/ReportBugRow.swift [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        let emailAddress = "ios@hedvig.com"
+        
+        row.keySignal.value = "Rapportera bugg"
+        row.valueSignal.value = emailAddress
+        
+        row.valueStyleSignal.value = .rowTitlePurple
+    
+        let device = Device()
+    
+        let deviceInfo = "Device: \(device)\nSystem: \(device.systemName) \(device.systemVersion)"
```
Comment (Iamsamwhoami) [inline Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/ReportBugRow.swift]: Can we add app-version and member id also?

File: Src/Screens/ProfileScreen/Screens/FeedbackScreen/Feedback.swift (hunk 1 above)
Comment (gustafguner) [reply Src/Screens/ProfileScreen/Screens/FeedbackScreen/Feedback.swift]: Yeah, that makes sense 👍

File: Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/ReportBugRow.swift (hunk 3 above)
Comment (gustafguner) [reply Src/Screens/ProfileScreen/Screens/FeedbackScreen/Viewables/ReportBugRow.swift]: Yes!

File: Src/Generic/Styling/TextStyle.swift [hunk 4]
```diff
@@ -60,7 +60,12 @@ extension TextStyle {
         style.font = HedvigFonts.circularStdBook!.withSize(17)
         style.color = .gray
     }
-
+    
+    static let rowTitlePurple = TextStyle.default.restyled { (style: inout TextStyle) in
```
Comment (Iamsamwhoami) [inline Src/Generic/Styling/TextStyle.swift]: I think I've created a textstyle that looks exactly like this already 🙈

Comment (Iamsamwhoami) [reply Src/Generic/Styling/TextStyle.swift]: `rowValueEditable` or maybe that that doesn't make sense

File: Src/Generic/Viewables/BackgroundWithLabel.swift [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+import Foundation
+import UIKit
+
+struct BackgroundWithLabel {
+    let labelText: DynamicString
+    let backgroundColor: UIColor?
+    let backgroundImage: UIImage?
+    let textColor: UIColor?
+    
+    init(labelText: DynamicString, backgroundColor: UIColor? = .purple, backgroundImage: UIImage? = nil, textColor: UIColor? = .white) {
```
Comment (Iamsamwhoami) [inline Src/Generic/Viewables/BackgroundWithLabel.swift]: Could you put each argument on it's separate row? 🙌

File: Src/Generic/Styling/TextStyle.swift (hunk 4 above)
Comment (gustafguner) [reply Src/Generic/Styling/TextStyle.swift]: Just changed the name to `rowValueLink`! 👍

File: Src/Generic/Viewables/BackgroundWithLabel.swift (hunk 5 above)
Comment (gustafguner) [reply Src/Generic/Viewables/BackgroundWithLabel.swift]: ✅

**Pull request merged**